pom.xml

```diff
@@ -22,6 +22,11 @@
             <artifactId>jersey-client</artifactId>
             <version>2.25</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
you might use maven <scope>test</scope> https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope